app/models/metric/ci_mixin/capture.rb

```diff
@@ -17,6 +17,20 @@ def queue_name_for_metrics_collection
     ems.metrics_collector_queue_name
   end
 
+  def split_capture_intervals(start_time, end_time)
+    intervals = []
+    start_hour = start_time
+    while start_hour != end_time
+      #TODO: replace 1.day with a configurable threshold defined by provider type
```

Comment:
Meh ... follow-up PR if anything.

It's possible that some providers will gather *so much* data every hour that attempting to even get just one day would be too much data.  Dunno.